pkg/controller/labelidentity/label_group_index.go

```diff
@@ -33,7 +33,9 @@ import (
 const (
 	// Cluster scoped selectors are stored under empty Namespace in indice.
 	emptyNamespace = ""
-	policyIndex    = "policyIndex"
+	// As defined in https://github.com/kubernetes/apimachinery/blob/master/pkg/labels/labels.go#L90
```

Comment:
you should use a permalink here

Reply:
+1, the permalink  is https://github.com/kubernetes/apimachinery/blob/3e2600dc79feea6cdc8a9224bc8a6a7fcfee1466/pkg/labels/labels.go#L90

Reply:
Done